### ISSUE TITLE
Change embeds to links

### ISSUE THREAD

**Issue by Carl-Robert** [SimonBrazell/privacy-redirect#42]
Hey,

Embeds on a private forum I use (XenForo 2.1 backend) need javascript, iframe etc tracking information to be whitelisted to load. Youtube embeds get changed to invidious embeds without allowing youtube javascript, but twitter and instagram embeds for example fail to load at all.

The redirecting works fine clicking on them once they've loaded, so I'd like to request an option to force embeds to display as links/clickable placeholders. This would also reduce bandwidth and power usage if not wanting to see the embeds.

**Comment by SimonBrazell** [comment]
Hi @Carl-Robert so you want me to add an option that halts youtube embed redirects but also stop it from loading normally? Twitter and Instagram embeds should both just load as normal I think (never tested it with a Twitter embed...) without redirection as Nitter and Bibliogram don't support embeds I believe.

**Comment by Carl-Robert** [comment]
No, I do not want embeds to work at all. I'd want an option to instead of loading embeds loading a placeholder (like twitter logo, instagram logo etc) or a plain link. Clicking these redirects normally to nitter etc. This way we can block twitter and instagram globally and not having to deal with embeds. XenForo is pretty popular backend and I bet more and more other forum software starts to support embeds from social media.

If outside the scope of this extension, I'd like an option to NOT process embeds (youtube only at the moment) so can use some other extension to turn embeds to links.

**Comment by SimonBrazell** [comment]
No I think it's within scope, and maybe something I would use myself actually. Should be relatively easy too I think, just redirect the embeds to a static HTML file stored within the extension that contains the redirect link. With an option to globally enable / disable this feature in the options menu.

Pretty busy lately with other work but I would definitely like to add a feature like this at some point.

**Comment by Victor239** [comment]
Twitter and YouTube embeds are big privacy leaks on some forums I browse, would be great to see this feature.
